ui/app/AppLayouts/Chat/ContactsColumn/EmptyView.qml

```diff
@@ -102,6 +102,7 @@ Item {
         Separator {
             anchors.topMargin: 0
             anchors.top: inviteFriendsContainer.bottom
+            color: Style.current.grey
```

Comment:
I think the color of the separator is ok as being `Style.current.border`. It's probably the rectangle itself that should change to  `Style.current.border` on line 18. What do you think?

Reply:
seems decent.